.github/workflows/nix.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        coq: [ "8.11", "8.10", "8.12" ]
+        coq: [ "8.11", "8.12" ]
```

Comment:
Should I simply add `"8.13"` here BTW?

Reply:
No, it requires an update of nixpkgs, but since I did a huge refactoring, it will not be compatible with the current configuration for mathcomp. Please leave it out, I will come back and fix it asap.

Reply:
Sure, thanks.